sdk/python/feast/permissions/auth_model.py

```diff
@@ -0,0 +1,15 @@
+from enum import Enum
```

Comment:
Pls see an example of decortors to add post-creation validation [here](https://vscode.dev/github/dmartinol/feast/blob/feast-rbac/sdk/python/feast/infra/offline_stores/contrib/trino_offline_store/trino.py#L100)